docs/madr/decisions/062-meshexternalservice-through-zone.md

```diff
@@ -0,0 +1,122 @@
+# MeshExternalService Accessible Only Through a Specific Zone
+
+* Status: accepted
+
+Technical Story: https://github.com/kumahq/kuma/issues/11071
+
+## Context and Problem Statement
+
+In certain cloud environments, mesh-enabled applications need to communicate with applications running in a Datacenter (DC) that are not part of the mesh. These DC applications are not exposed to the internet. To address this, we installed a ZoneCP in the DC. This setup allows DC applications to be resolved from the Kuma Egress running in the DC.
```

Comment:
These sentences are from users, right? It's better to convert the way to be from our (Kuma authors') perspective?

Reply:
Changed a bit